plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/BaseTestMySqlTableStatisticsTest.java

```diff
@@ -81,13 +81,9 @@ protected QueryRunner createQueryRunner()
 
     @Override
     @Test
-    @Flaky(
-            // TODO replace @Flaky with assertEventually or @Flaky-like annotation for same purpose
-            issue = "TODO",
-            match = "Expecting.*to be close to|ComparisonFailure.*NDV for")
     public void testNotAnalyzed()
     {
-        String tableName = "test_not_analyzed";
+        String tableName = "test_not_analyzed_" + randomTableSuffix();
```

Comment:
Nit: extract randomized table names to separate commit

Reply:
And add reasoning in commit message + PR description since it won't be obvious to other people who don't have the context.

Reply:
👍

Reply:
@hashhar - see updated PR Title + Description (separated into 2 commits - 2nd commit removes @Flaky + disables `testNotAnalyzed` - 1st commit is preparatory in case any tests fail in future / we can re-run them with unique table names, etc - let me know if the description / commit messages could do with some work - thank you!